lib/src/views/Calendar/FadeTransitionGroup.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import { makeStyles } from '@material-ui/core/styles';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+
+interface FadeTransitionProps {
+  transKey: React.Key;
+  className?: string;
+  children: React.ReactChild;
+}
+
+const animationDuration = 500;
```

Comment:
Should we use a transition duration from on the of the specification (theme.transitions.duration.x)?

Reply:
```
shortest: 150
shorter: 200
short: 250
standard: 300
complex: 375
enteringScreen: 225
leavingScreen: 195
```

All these durations are too fast for this animation.